get_subscription.py

```diff
@@ -66,13 +66,15 @@
 
 def mount_get_root(mount):
     drive = mount.get_drive()
+    if not drive or not drive.is_removable():
+        return False
```

Comment:
I would return `None` here, since the function otherwise returns an object, and we probably want to keep the types compatible.